crates/sui-node/src/admin.rs

```diff
@@ -30,11 +30,16 @@ use tracing::info;
 // Vote to close epoch 2 early
 //
 //   $ curl -X POST 'http://127.0.0.1:1337/force-close-epoch?epoch=2'
+//
+// View current authorities capabilities:
```

Comment:
not plural

Reply:
it is plural - it returns all capabilities for all authorities that have been received by the current authority. I will clarify the comment.